ui/tests/integration/components/job-editor-test.js

```diff
@@ -127,7 +127,6 @@ module('Integration | Component | job-editor', function (hooks) {
 
     await renderNewJob(this, job);
     await planJob(spec);
-    console.log('wait');
```

Comment:
I don't think this is required here. I'll tackle it since my work is close to this to prevent a merge conflict.